.github/workflows/os_x_staticbuild.yml

```diff
@@ -10,16 +10,16 @@ jobs:
         uses: actions/checkout@v2
       - name: Install Dependencies
         run: |
-          brew install nasm automake ninja libtool cmake pkgconfig protobuf
+          brew install nasm automake ninja libtool cmake pkgconfig protobuf hdf5 zlib
```

Comment:
Why are the dependencies on hdf5 zlib introduced when switching to pytest?

Reply:
this was for installing h5py and pillow, which triggers compile in source distribution.